beancount/scripts/deps.py

```diff
@@ -125,7 +125,7 @@ def check_python_magic():
             # 'filemagic' is installed; install python-magic.
             raise ImportError
         return ('python-magic', 'OK', True)
-    except ImportError:
+    except (ImportError, OSError):
         return ('python-magic', None, False)
```

Comment:
Why does an import result in an OSError? Can you please submit this patch as a standalone PR?

Reply:
Exactly the error in #523

Reply:
See my comments in #523